src/illus/nocreations.svg

```diff
@@ -0,0 +1,14 @@
+<svg width="176" height="128" viewBox="0 0 176 128" fill="none" xmlns="http://www.w3.org/2000/svg">
+<rect y="7.05518" width="168.239" height="120.945" rx="6" fill="#F7F8FB"/>
```

Comment:
In order to support the night-mode, `fill` attribute should include css variables: `fill="var(--athens,#F7F8FB")`